nrf_security/src/psa_crypto_driver_wrappers.c

```diff
@@ -716,15 +716,11 @@ psa_status_t psa_driver_wrapper_import_key(
                return( status );
 #endif /* PSA_CRYPTO_DRIVER_HAS_ACCEL_KEY_TYPES_OBERON */
 #endif /* PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT */
-            /* Fell through, meaning no accelerator supports this operation */
-#if defined(MBEDTLS_PSA_BUILTIN_HAS_KEY_TYPE)
```

Comment:
Why not add enabled nrf_oberon PSA Crypto driver as a matching criteria? This detail gets lost in a commit in history...

Reply:
I prefer to have that as default in the core in all cases. We can try to remove it from the CC3XX driver as well as it doesn't add any value I think.